pennylane_lightning/core/_measurements_base.py

```diff
@@ -130,24 +130,32 @@ def expval(self, measurementprocess: MeasurementProcess):
             measurementprocess.obs.name, measurementprocess.obs.wires
         )
 
+    @abstractmethod
+    def _probs_retval_conversion(self, probs_results: Any) -> np.ndarray:
+        """Convert the data structure from the C++ backend to a common structure through lightning devices."""
+
```

Comment:
What about writing a default implementation for this method to avoid rewriting the logic in LQ and LK?
```suggestion
    def _probs_retval_conversion(self, probs_results: Any) -> np.ndarray:
        """Convert the data structure from the C++ backend to a common structure through lightning devices.
        Args:
            probs_result (Any): Result provided by C++ backend.
        Returns:
            np.ndarray with probabilities of the supplied observable or wires.
        """
        return probs_results;
```